README.md

```diff
@@ -76,29 +65,39 @@ Then you can use it like so: `npm run cmp`.
 }
 ```
 
-## Develop
+Read more on migration syntax on https://github.com/contentful/contentful-migration
+
+### Available commands
+
+
+#### `cmp generate <migrationName>`
+Generates a migration with the given name and a timestamp prepended ex: YYYYMMDDhhmmssxxx-add-user-type.js.
+
+#### `cmp migrate`
+Applies all _up_ operations of the non applied migrations to the CTF_ENVIRONMENT set in the `.env` file
+
+#### `cmp rollback`
+Applies the _down_ operations (i.e rolls back ) of the latest migration batch
+
+#### `cmp aux:create <name?>`
+Creates an aux environment based on CTF_ENVIRONMENT. You can give it an optional name.
+
+#### `cmp aux:drop <name>`
+Drop the environment with the given name
+
+#### `cmp aux:test`
+Creates environment from CTF_ENVIRONMENT, applies new migrations, and drops the environment
+
+
+## For development of the CMP tool
 
 1. Clone this project
-2. run npm i
-3. run npm link
-4. create a test project in another folder and cd into it
-5. run npm init
-6. run npm i @prototyp-stockholm/contentful-migrator-programme
-7. run npm link @prototyp-stockholm/contentful-migrator-programme
+2. `run npm i`
+3. `run npm link`
+4. Create a test project in another folder and cd into it
+5. `run npm init`
+6. `run npm i @prototyp-stockholm/contentful-migrator-programme`
+7. `run npm link @prototyp-stockholm/contentful-migrator-programme`
```

Comment:
```suggestion
2. run `npm i`
3. run `npm link`
4. Create a test project in another folder and cd into it
5. run `npm init`
6. run `npm i @prototyp-stockholm/contentful-migrator-programme`
7. run `npm link @prototyp-stockholm/contentful-migrator-programme`
```

Reply:
I'd rather skip the 'run' altogether. What do you think?